### DISCUSSION TITLE
Plot dot circle legend

### DISCUSSION THREAD

**Discussion by arky** [observablehq/plot#807] [Q&A] [answered]
Is it possible to add circle size scale legend. At the moment, its seems only color legends are possible?

Am trying to add circle legend scale to the following chart created with plot.dot:

![image](https://user-images.githubusercontent.com/808895/158129018-e4cb6079-bc7a-4e9c-964b-ed6e3fa713de.png)

https://observablehq.com/@arky/disasters-in-south-eastern-asia-1900-2021

**Comment by Fil** [comment, marked as answer]
Here's a suggestion https://observablehq.com/compare/a1aebf3714a2c506...14e4d8b8064c721b

It is based on the code in the pull request https://github.com/observablehq/plot/pull/665 ; relevant issue: https://github.com/observablehq/plot/issues/236.


<img width="927" alt="Capture d’écran 2022-03-14 à 09 22 27" src="https://user-images.githubusercontent.com/7001/158132701-90579fac-9f2b-47b6-a06c-ab9f91dabe8b.png">

Please feel free to play with the code, comment and suggest.

**Reply by arky** [reply]
Thank you @Fil!
